general/clients/css-customization.md

```diff
@@ -86,6 +86,32 @@ This will affect the played/watched indicator. Replace the hex color with any va
 
 ![Screenshot of watched indicators with a custom transparent color applied](~/images/custom-css-transparentwatched.png)
 
+### Remove Home Icon from Header
+
+```css
+.headerHomeButton { display: none; }
+
+.headerButton.headerButtonRight.headerUserButton.paper-icon-button-light { display: none; }
+```
+
+### Remove Cast Icon from Header
+
+```css
+.headerCastButton { display: none; }
+```
+
+### Remove Sync Icon from Header
+
+```css
+.headerSyncButton { display: none; }
+```
+
+### Remove User Settings from Header
+
+```css
+.material-icons.person { display: none; }
```

Comment:
I _think_ we use this icon in more places and this CSS will remove those instances. Did you test that?

Reply:
I didn't see anything abnormal, you have any idea where?

Reply:
At least the menu when you click on your avatar, but I think some places in the dashboard too

Reply:
All looks good. I didn't see anything on the dashboard

Reply:
![image](https://user-images.githubusercontent.com/2305178/103895299-7d549380-50f0-11eb-9a83-8bdcaffc879f.png)